docs/index/mv-guide.md

```diff
@@ -217,40 +217,72 @@ release, user can do as following:
 Basically, user can manually trigger the operation by re-building the datamap.
 
 ## MV TimeSeries Support
-MV non-lazy datamap supports TimeSeries queries. Supported granularity strings are: year, month, week, day,
-hour,thirty_minute, fifteen_minute, ten_minute, five_minute, minute and second.
+Time series data are simply measurements or events that are tracked, monitored, downsampled, 
+and aggregated over time. Materialized views with automatic refresh mode supports TimeSeries queries.
 
- User can create MV datamap with timeseries queries like the below example:
+Carbondata provides built-in time-series udf with the below definition.
+
+```timeseries(time_series_column, 'granularity')```
+
+Columns provided in timeseries udf should be of TimeStamp/Date type.
+
+Below table describes the time hierarchy and levels that can be provided in a time-series udf, 
+so that it supports automatic roll-up in time dimension for query.
+
+| Granularity | Description |
+|---------------|----------------|
+| Year | Data will be aggregated over year |
+| Month |  Data will be aggregated over month  | 
+| Week |  Data will be aggregated over week |
+| Day |  Data will be aggregated over day |
+| Hour |  Data will be aggregated over hour |
+| Thirty_minute |   Data will be aggregated over every thirty minutes |
+| Fifteen_minute |   Data will be aggregated over every fifteen minutes  |
+| Ten_minute |  Data will be aggregated over every ten minutes  |
+| Five_minute | Data will be aggregated over every five minutes|
+| Minute |  Data will be aggregated over minute|
+| Second |  Data will be aggregated over seconds|
+
+Timeseries udf having column as Date type support's only year, month, day and week granularities.
+
+Users can create materialized views with timeseries queries like the below example:
 
   ```
-  CREATE DATAMAP agg_sales
-  ON TABLE sales
-  USING "MV"
+  CREATE materialized view agg_sales
   AS
     SELECT timeseries(order_time,'second'),avg(price)
     FROM sales
     GROUP BY timeseries(order_time,'second')
   ```
-Supported columns that can be provided in timeseries udf should be of TimeStamp/Date type.
-Timeseries queries with Date type support's only year, month, day and week granularities.
+And execute the below query to check timeseries data
```

Comment:
Please add:
In this example, a materialized view of aggregated table on price column will be created, which will be aggregated on every 1 second. 

suggest to change this example to 1 minute for more real case

Reply:
It is better to add an example table with values to show how the MV will look like. It can help reader understand how  it works